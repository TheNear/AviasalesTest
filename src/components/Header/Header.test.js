import React from "react";
import { shallow } from "enzyme";
import Header from "./Header";

describe("Test Header component", () => {
  it("Render empty Header", () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
