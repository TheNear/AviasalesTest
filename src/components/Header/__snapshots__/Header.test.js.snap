// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Header component Render empty Header 1`] = `
<header
  className="header"
>
  <img
    alt="Site logo"
    className="logo"
    src="logo.svg"
  />
</header>
`;
